slcshelterarrests/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java

```diff
@@ -99,24 +98,27 @@ public static void main( String[] args ) throws InterruptedException {
     // CUSTOM FUNCTIONS DEFINED BELOW
 
     public static String paddedTime( String time ) {
-      int original = Integer.parseInt(time);
-      String padded = String.format("%04d", original);
-      return padded;
+        int original = Integer.parseInt( time );
+        String padded = String.format( "%04d", original );
+        return padded;
     }
 
     public static String standardizeDateTime( Object myDate, Object myTime ) {
-      if (myDate != null && myTime == null) {
-        FormattedDateTime date = new FormattedDateTime( myDate.toString(), null, "MM/dd/yyyy", null);
-        return date.getDateTime();
-      }
-      if (myDate == null && myTime != null) {
-        FormattedDateTime time = new FormattedDateTime( null, paddedTime(myTime.toString()), null, "HHmm");
-        return time.getDateTime();
-      }
-      if (myDate != null && myTime != null) {
-        FormattedDateTime dateTime = new FormattedDateTime( myDate.toString(), paddedTime(myTime.toString()), "MM/dd/yyyy", "HHmm");
-        return dateTime.getDateTime();
-      }
-      return null;
+        if ( myDate != null && myTime == null ) {
```

Comment:
The custom functions defined below the integration are specific to the dataset that was integrated (since it might involve custom parsing malformed dates in the dataset). The FormattedDateTime class it uses should be the same across all the PRs.